contrib/grinder/README.md

```diff
@@ -1,99 +1,5 @@
 # Blueflood Grinder Integration
-##Intro
-Grinder is a distributed load testing tool described [here](http://grinder.sourceforge.net/g3/getting-started.html)
 
-This code defines implementations of grinder worker threads meant to repeatedly invoke the required number of BF api calls during each "reporting interval"
+### DEPRECATED
```

Comment:
@ChandraAddala This is an open-source repo. Open-source users cant see our internal wiki. Were you thinking the github wiki instead?